rueppellii-lvlup-asp.net-core/Models/User.cs

```diff
@@ -19,6 +19,8 @@ public class Users
 
         public string Pic { get; set; }
 
-        public List<Badges> Badges { get; set; }
+        public List<Review> Reviews { get; set; }
+        public List<Pitch> Pitches { get; set; }
+        public List<UserLevel> UserLevels { get; set; }
```

Comment:
According to the drawn DB scheme (https://app.sqldbm.com/MySQL/Share/m3vZyWVhNI3ZPAnWPSs_GEGFrngIE8md_DYjF4jNYw0) Users should have Badges not a combination of Reviews, Pitches and UserLevels. Is this smth I have missed and you agreed to do it like this?

Reply:
Resolved.